service/Dockerfile

```diff
@@ -2,10 +2,12 @@
 FROM openjdk:8-jre
 
 WORKDIR /usr/local/src
-COPY . /usr/local/src
+COPY . /usr/local/src/
```

Comment:
`src` contains the static assets served by jetty. Nothing in`target` needs to be at `/usr/local/src`, which is why it gets deleted. `setup.py` was in the original image, so I figured it would need to stay.